README.md

```diff
@@ -78,6 +79,30 @@ in all your `CREATE JAVA ADAPTER SCRIPT` statements. The old entry point was rem
 * Allows redirecting log output to a remote machine
 * Allows remote debugging with the Java Debugger
 
+## Supported Data Sources
```

Comment:
This looks like a duplication of the section above.  We should have only one of them left

Reply:
Maybe this table is more readable

Reply:
yes, totally. I just put it to have your feedback.